plugins/tutor-contrib-library-authoring-mfe/tutor_library_authoring_mfe/plugin.py

```diff
@@ -0,0 +1,98 @@
+    [
+        ("LIBRARY_AUTHORING_MICROFRONTEND_VERSION", __version__),
+        ("LIBRARY_AUTHORING_MFE_APP", {
+            "name": "library-authoring",
+            "repository": "https://github.com/openedx/frontend-app-library-authoring",
+            "port": 3001,
+        })
+    ]
+)
+
+tutor_hooks.Filters.CONFIG_UNIQUE.add_items(
+    [
+        ("BLOCKSTORE_COLLECTION_UUID", str(uuid.uuid4()))
+    ]
+)
+
+
+########################################
+# INITIALIZATION TASKS
+########################################
+
+MY_INIT_TASKS: list[tuple[str, tuple[str, ...]]] = [
+    ("cms", ("library_authoring_mfe", "jobs", "init", "cms.sh")),
+]
+
+# For each task added to MY_INIT_TASKS, we load the task template
+# and add it to the CLI_DO_INIT_TASKS filter, which tells Tutor to
+# run it as part of the `init` job.
+for service, template_path in MY_INIT_TASKS:
+    full_path: str = pkg_resources.resource_filename(
+        "tutor_library_authoring_mfe", os.path.join("templates", *template_path)
+    )
+    with open(full_path, encoding="utf-8") as init_task_file:
+        init_task: str = init_task_file.read()
+    tutor_hooks.Filters.CLI_DO_INIT_TASKS.add_item((service, init_task))
+
+########################################
+# TEMPLATE RENDERING
+########################################
+
+tutor_hooks.Filters.ENV_TEMPLATE_ROOTS.add_items(
+    # Root paths for template files, relative to the project root.
+    [
+        pkg_resources.resource_filename("tutor_library_authoring_mfe", "templates"),
+    ]
+)
+
+########################################
+# PATCH LOADING
+########################################
+
+# For each file in tutor_library_authoring_mfe/patches,
+# apply a patch based on the file's name and contents.
+for path in glob(
+    os.path.join(
+        pkg_resources.resource_filename("tutor_library_authoring_mfe", "patches"),
+        "*",
+    )
+):
+    with open(path, encoding="utf-8") as patch_file:
+        tutor_hooks.Filters.ENV_PATCHES.add_item((os.path.basename(path), patch_file.read()))
+
+# Tutor overwrites webpack.dev.config.js, but this MFE depends on some code
+# in that file to work correctly so we have to restore it here manually.
+# https://github.com/openedx/frontend-app-library-authoring/blob/b95c198b/webpack.dev.config.js
+tutor_hooks.Filters.ENV_PATCHES.add_item(("mfe-webpack-dev-config","""
+const fs = require('fs');
+
+// If this is the Library Authoring MFE, apply this fix:
+if (fs.existsSync("src/library-authoring/edit-block/LibraryBlock/xblock-bootstrap.html")) {
+    const path = require('path');
+    const CopyWebpackPlugin = require('copy-webpack-plugin');
+    module.exports = merge(module.exports, {
+    plugins: [
+        new CopyWebpackPlugin({
+        patterns: [{
+            context: path.resolve(__dirname, 'src/library-authoring/edit-block/LibraryBlock'),
+            from: 'xblock-bootstrap.html',
+        }],
+        }),
+    ],
+    });
+}
+"""))
```

Comment:
I haven't checked: can we also do this [for prod](https://github.com/brian-smith-tcril/frontend-app-library-authoring/blob/62468ea8c19a81a052034bf403dc5b13597401a4/webpack.prod.config.js)?  Or do we have to have the file live in the MFE?

Reply:
I don't fully remember the details of this. Should we consider this a blocker or would it be ok to make a new issue for discovery/cleanup?

Reply:
Not a blocker for this.  Since I have my hands dirty already, I can do some testing myself!

Reply:
Since this was merged without that changing should I make a discovery issue?